database/index.js

```diff
@@ -23,8 +23,50 @@ const dbConnection = mysql.createConnection(mysqlConfig);
 
 const getAllExpReviews = (expId, callback) => {
   const query = 'select * from reviews where experience_id = ?';
-  const experienceId = expId;
-  dbConnection.query(query, experienceId, (err, response) => {
+  dbConnection.query(query, expId, (err, response) => {
+    if (err) {
+      callback(err);
+    } else {
+      callback(null, response);
+    }
+  });
+};
+
+const createExpReviews = (expId, reviewsData, callback) => {
```

Comment:
It looks like this does one review at a time? (For mine, I decided that the CRUD operations worked on an entire expId at once -- so you have to pass in an array of subrecords. I have no idea if the 'subrecords one at a time' or 'subrecords many at a time' is preferred. I just thought since GET returned an array of subrecords, my CREATE would take an array of subrecords.)

On the other hand, I may not be understanding line 36, that says 'const reviews = { ...reviewsData}' and how that ties in with line 39 'dbConnection.query(query, reviews, ...)'

Reply:
It does handle one new post per experience ID

line 36 is to copt the reviewsData, and in the next line include the expID into the copied obj. When you have set you can pass in an obj